modules/HapticGlove_module/include/ControlHelper.hpp

```diff
@@ -37,6 +37,8 @@ Eigen::Map<Eigen::VectorXd> toEigenVector(std::vector<double>& vec);
 
 Eigen::Map<Eigen::VectorXd> toEigenVector(yarp::sig::Vector& vec);
 
+Eigen::Map<const Eigen::VectorXd> toEigenVector(const yarp::sig::Vector& vec);
```

Comment:
Interestingly, this conversion should be there also in yarp: https://yarp.it/git-master/namespaceyarp_1_1eigen.html#aa4598147bb94b1a7d4e5c336a9351f9b

Not sure when it has been introduced